pom.xml

```diff
@@ -335,6 +336,11 @@
           <version>2.17</version>
           <dependencies>
             <dependency>
+              <groupId>org.apache.apex</groupId>
+              <artifactId>apex-codestyle-config</artifactId>
+              <version>${project.version}</version>
```

Comment:
This is a dependency on SNAPSHOT version that will be propagated to Malhar library. IMO, codestyle artifact should be on a different release cycle compared to Apex and/or Malhar and this is where separating it into a separate project will help. This will also help with avoiding every developer rebuilding the artifact every time Apex is built.